core/chains/evm/txmgr/eth_resender.go

```diff
@@ -30,13 +33,14 @@ const defaultResenderPollInterval = 5 * time.Second
 // can occasionally be problems with this (e.g. abnormally long block times, or
 // if gas bumping is disabled)
 type EthResender struct {
-	orm       ORM
-	ethClient evmclient.Client
-	ks        txmgrtypes.KeyStore[common.Address, *big.Int, int64]
-	chainID   big.Int
-	interval  time.Duration
-	config    Config
-	logger    logger.Logger
+	orm                 ORM
+	ethClient           evmclient.Client
+	ks                  txmgrtypes.KeyStore[common.Address, *big.Int, int64]
+	chainID             big.Int
+	interval            time.Duration
+	config              Config
+	logger              logger.Logger
+	lastAlertTimestamps map[common.Address]time.Time
```

Comment:
after my PR recently merged, you no longer will have access to common.Address here.
Will hv to change this key to map[string]time.Time
And get the string from: enabledAddress.String().